scripts/flake8_py_version_check.py

```diff
@@ -0,0 +1,36 @@
+import json
+import re
+import subprocess
+
+
+def main():
+    # find a line in pyproject.toml that looks like `python-requires = "..."`
+    # with forgiving spacing and single or double quotes
+    with open("pyproject.toml") as fp:
+        pyproject_txt = fp.read()
+    match = re.search(r"""\n\s*requires-python\s*=\s*['"](.*)['"]""", pyproject_txt)
+    assert match is not None, "'requires-python' line not found in pyproject.toml"
+
+    flake8_bugbear_requires = match.group(1).strip()
+
+    # get pypi data for flake8 as json
+    curl_output = subprocess.getoutput(
+        "curl -L -s --header 'Accept: application/vnd.pypi.simple.v1+json'"
```

Comment:
Nice use of PEP691 ... Glad to see it getting used :)

But lets just use latest for version here so we just move forward with the API moving (which will be slow moving ...) and if it breaks I/someone can fix the code ...
```suggestion
        "curl -L -s --header 'Accept: application/vnd.pypi.simple.latest+json'"
```